iommu.c

```diff
@@ -122,7 +130,7 @@ u32 iommu_load_device_table(u32 cap, volatile u64 *completed)
     print("IOMMU_MMIO_EVENT_LOG_BA\n");
 
     /* Clear EventLogInt set by IOMMU not being able to read command buffer */
-    mmio_base[IOMMU_MMIO_STATUS_REGISTER] &= ~2;
+    mmio_base[IOMMU_MMIO_STATUS_REGISTER] &= IOMMU_SR_EventLogInt;
```

Comment:
`~` got lost.

Reply:
No, it actually shouldn't be there in the first place. This bit is write-1-to-clear.

Reply:
Maybe update comment to say that it's write-1-to-clear?  I read about it in PR's description, but some time later saw "clear" and `&=` without `~` and assumed the code is wrong.

Reply:
https://github.com/TrenchBoot/secure-kernel-loader/compare/43049429092897dd3fe6663eb9c5329f12f012b1..d2cb1b776cd1c44abedbd0f68401d62c46b6e05a